internal/btf/types.go

```diff
@@ -134,6 +152,14 @@ func (p *Pointer) copy() Type {
 	return &cpy
 }
 
+func (p *Pointer) raw(strings *stringTable) rawType {
+	var raw rawType
+	raw.SetKind(kindPointer)
+	raw.SizeType = uint32(p.Target.ID())
```

Comment:
That's a good idea, but I'm not sure how/where to do that. At this point, since we're converting to raw, we need to have an ID. Unfortunately, only something outside an individual type would have knowledge of the already consumed type IDs to be able to assign them without potential for duplicates, so they must be done elsewhere before conversion to raw. (Or conversion to raw would need to be disallowed on an individual `Type` and only on, say, a whole spec.)

Reply:
Will keep this in mind for a future PR to reintroduce this functionality, but, as discussed, this is not needed for the functionality this PR is trying to introduce so it has been removed.